### ISSUE TITLE
NULLs are not returned in a list

### ISSUE THREAD

**Issue by mdwaud** [sunng87/datafusion-postgres#5]
I have a list where some elements may be `NULL`. Unfortunately they aren't showing up in the return, for example:

In postgres

```
postgres=# SELECT array[1,NULL,3];
   array    
------------
 {1,NULL,3}
(1 row)
```

In datafusion

```
datafusion=> SELECT array[1,NULL,3];
 make_array(Int64(1),NULL,Int64(3)) 
------------------------------------
 [1,,3]
(1 row)
```

**Comment by mdwaud** [comment]
Hmmm... `array_length` is telling me it's there, though:

In postgres
```
postgres=# SELECT ARRAY_LENGTH(ARRAY[1, NULL], 1);
 array_length 
--------------
            2
(1 row)
```

In datafusion-postgres
```
datafusion=>  SELECT ARRAY_LENGTH(ARRAY[1, NULL], 1);
 array_length(make_array(Int64(1),NULL),Int64(1)) 
--------------------------------------------------
                                                2
(1 row)
```

Let me see if I can come up with a better example to reproduce what I'm seeing. For context, I have a column in data fusion where a value may be `None`. In psql I am seeing an empty list when the value is `None`. It's possible there's an error in my setup.

**Comment by sunng87** [comment]
Yeah, I think it's just how pgwire serialize `NULL` value. It should be text `NULL` instead of empty string.

**Comment by sunng87** [comment]
Just learned that postgres only uses NULL for NULL value in array (and other compound data strcutures maybe)

**Comment by sunng87** [comment]
Thank you for pointing out. I added another fix for `to_sql_text` for `bool`.

**Comment by sunng87** [comment]
I just released a new version of pgwire to include this fix. You can also test latest HEAD from this repo

**Comment by mdwaud** [comment]
Thanks for the quick response!
